### PR TITLE
Auto OCP-29693 OCP-29706

### PR DESCRIPTION
**Pull request by xiuwang**
@openshift/devexp-qe Please help review, thanks
Passed locally http://pastebin.test.redhat.com/871733

### CODE REVIEW

File: features/registry/registry.feature [hunk 1]
```diff
@@ -173,3 +173,68 @@ Feature: Testing registry
       | Mirroring completed in |
     Given the "myimage" image stream was created
     And the "myimage" image stream becomes ready
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-29693
+  @admin
+  Scenario: [Disconnect]Import image from a secure registry using node credentials
+    Given I have a project
+    And evaluation of `image_content_source_policy('image-policy-aosqe').mirror_registry(cached: false, user: admin)` is stored in the :mirror_registry clipboard
```
Comment (liangxia) [inline features/registry/registry.feature:182]: We do not need `user: admin` there.

File: features/registry/registry.feature [hunk 2]
```diff
@@ -173,3 +173,68 @@ Feature: Testing registry
...
+    When a pod becomes ready with labels:
+      | deployment=mysql-1 |
+    When I run the :describe client command with:
+      | resource | pod                | 
+      | l        | deployment=mysql-1 |
+    And the output should match:
+      | Successfully pulled image "image-registry.openshift-image-registry.svc:5000/<%= project.name %>/mysql | 
+    When I run the :import_image client command with:
+      | from       | <%= cb.mirror_registry %>rhscl/ruby-25-rhel7:latest |
+      | confirm    | true                                                |
```
Comment (liangxia) [inline features/registry/registry.feature:201]: Align issue.

Comment (xiuwang) [reply features/registry/registry.feature:201]: It keeps same space with next line.

File: features/registry/registry.feature (hunk 1 above)
Comment (xiuwang) [reply features/registry/registry.feature:182]: @liangxia Updated as comment, and it passed.

File: features/registry/registry.feature (hunk 2 above)
Comment (liangxia) [reply features/registry/registry.feature:228]: Checked in another computer, the align is good.

**Pull request merged**